doc/nrf/gs_programming.rst

```diff
@@ -122,3 +122,92 @@ After completing the :ref:`manual <build_environment_cli>` or :ref:`automatic <g
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+| :file:`matter.ota`              | :ref:`ug_matter`-specific OTA image that contains a Matter-compliant header                     | DFU over Matter for both single-core and multi-core applications.        |
+|                                 | and a DFU multi-image package that bundles user-selected firmware images.                       |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+| :file:`<file_name>.zigbee`      | :ref:`ug_zigbee`-specific OTA image that contains the Zigbee application                        | DFU over Zigbee for both single-core and multi-core applications         |
+|                                 | with the Zigbee OTA header used for providing information about the image to the OTA server.    | in the |NCS| v2.0.0 and later.                                           |
+|                                 | The *<file_name>* includes manufacturer's code, image type, file version, and comment           |                                                                          |
+|                                 | (customizable by user, sample name by default).                                                 |                                                                          |
+|                                 | For example: :file:`127F-0141-01020003-light_switch.zigbee`.                                    |                                                                          |
++---------------------------------+-------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------+
+
+.. output_build_files_table_end
+
+.. _gs_programming_output_other_files:
+
+Other output build files
+========================
+
+The following table lists secondary build files that can be generated when building firmware, but are only used to create the final output build files listed in the table above.
+
++---------------------------------+----------------------------------------------------------------------------------------------------------+
+| File                            | File format                                                                                              |
++=================================+==========================================================================================================+
+| :file:`zephyr.elf`              | A metadata file for the image that is being built. Can be used for debugging purposes.                   |
```

Comment:
I wouldn't call an ELF file a meta data file.

It's the most complete file generated by the system, and only because our flash tools doesn't support flashing of ELF files directly it must be converted. (Plus the fact that we might manipulate / merge hex files, but that's a different story)
Some tools actually supports flashing of ELF files.